packages/pg-v5/test/lib/psql.js

```diff
@@ -29,37 +37,48 @@ const bastionDb = {
 }
 
 describe('psql', () => {
+  let fakePsqlProcess, fakeTunnel, tunnelStub
+  let sandbox
+  let mockSpawn
+  let psql, bastion
+
   beforeEach(() => {
-    sinon.stub(Math, 'random').callsFake(() => 0)
+    sandbox = sinon.createSandbox()
+    tunnelStub = sandbox.stub().callsFake((_config, callback) => {
+      fakeTunnel = new TunnelStub()
+      callback(null, fakeTunnel)
+    })
+    mockSpawn = createSpawnMocker(sandbox)
+    bastion = proxyquire('../../lib/bastion', {
+      'tunnel-ssh': tunnelStub
+    })
+    psql = proxyquire('../../lib/psql', {
+      './bastion': bastion
+    })
+    fakePsqlProcess = new FakeChildProcess()
+    sandbox.stub(Math, 'random').callsFake(() => 0)
   })
 
-  afterEach(() => {
-    Math.random.restore()
```

Comment:
this was removed since the sandbox gets reset between tests and will do the restore for us since the stub was created through the sandbox. https://sinonjs.org/releases/latest/sandbox/